core-tests/src/test/java/org/mule/runtime/core/routing/AsynchronousUntilSuccessfulProcessingStrategyTestCase.java

```diff
@@ -120,6 +120,14 @@ public void setUp() throws Exception {
                                                                                                           .build());
   }
 
+  @After
+  public void after() {
```

Comment:
That's only the case for AbstractMuleContextTestCase. This text extends from AbstractMuleTestCase and creates the service within itself.